hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -544,15 +543,40 @@ public static Option<Pair<Integer, String>> getLatestLogVersion(FileSystem fs, P
   }
 
   /**
-   * computes the next log version for the specified fileId in the partition path.
+   * Get the latest log version for the fileId in the partition path.
+   */
+  public static int getLatestLogVersion(FileSystem fs, Path partitionPath, final String fileId,
```

Comment:
fix formatting?

Reply:
Removed because it is not used anywhere.